LIPs/LIP-36.md

```diff
@@ -78,7 +81,9 @@ Add the following steps to the algorithm for `bondingManager.updateTranscoderWit
 
 - If the orchestrator has not called reward in the current round, set `activeCumulativeRewards = cumulativeRewards`
 - Let `X` be the orchestrator's active stake for the current round
-- Let `earningsPool` be orchestrator's earnings pool for the current round and `prevEarningsPool` be orchestrator's earnings pool for the previous round
+- Let `earningsPool` be orchestrator's earnings pool for the current round and `prevEarningsPool` be the orchestrator's earnings pool for the previous round. 
+    - If `prevEarningsPool.cumulativeRewardFactor == 0` set it to the `cumulativeRewardFactor` of the `earningsPool` for the orchestrator's `lastRewardRound`.
```

Comment:
State updates aren't actually necessary here right since the calculation can just reference the earningsPool for the lastRewardRound and lastFeeRound without updating prevEarningsPool?

Reply:
fixed in 4823679